src/api/common/utils/Utils.js

```diff
@@ -511,4 +511,16 @@ export function isPremiumLegacyCustomizationEnabled(customer: Customer): boolean
 
 export function isCustomizationEnabledForCustomer(customer: Customer, feature: FeatureTypeEnum): boolean {
 	return !!customer.customizations.find(customization => customization.feature === feature)
+}
+
+/**
+ * If val is non null, returns the result of val passed to action, else null
+ * @param val
+ * @param action
+ * @returns {U|null}
+ */
+export function ifNotNull<T, U>(val: ?T, action: T => U): U | null {
```

Comment:
Questionable.
I would at least name it `mapNullable()` Or something like that?
And get rid of useless jsdoc lines for arguments and return

Reply:
I can rename, I find it reads a lot better than lots of dangling `: null`s which i think is the only reason to use